sites/geohub/src/components/controls/vector-styles/PropertySelect.svelte

```diff
@@ -47,7 +47,7 @@
 <div class="control has-icons-left">
   <div
     style="margin-right: 2%"
-    class="select is-flex is-justify-content-left select is-small">
+    class="select is-flex is-justify-content-left select is-normal">
```

Comment:
Are you sure bulma has is-normal class? I don't think there is such a class name for bulma.